docker-entrypoint.sh

```diff
@@ -588,13 +591,17 @@ _main() {
 			sleep 5
 		done
 	else
-		if [ "x${PGAUTO_ONESHOT}" = "xyes" ]; then
-			echo "*****************************************************************************************************"
-			echo "'One shot' automatic upgrade was requested, so exiting now rather than starting the PostgreSQL server"
-			echo "*****************************************************************************************************"
-		else
-			exec "$@"
+		# If the upgrade process ran, then we need to launch the post-upgrade script in the background
+		if [ "${UPGRADE_PERFORMED}" -eq 1 ]; then
+			TIMESTAMP_NOW=$(date +'%Y.%m.%d-%H.%M')
+			echo "**********************************************************************************************************************"
+			/usr/local/bin/pgautoupgrade-postupgrade.sh "${PGDATA}" "${PGAUTO_ONESHOT}" 2>&1 | tee "${PGDATA}/${TIMESTAMP_NOW}-pgautoupgrade.log" &
+			echo "Post upgrade script launched, with output being saved to ${PGDATA}/${TIMESTAMP_NOW}-pgautoupgrade.log in the container"
```

Comment:
I wonder if we even have to duplicate the output into a separate log file -> usually in Docker everything is written to stdout.

Reply:
Yeah, I'm not sure either.  It figure it might be useful to keep around just in case it turns out to be useful down the road for users.  ie months after an upgrade

It shouldn't hurt anyway. :smile:

Reply:
Thinking over this more, I'll just get rid of it the separate log file.  Should be simpler that way. :smile:

Reply:
Done.